config/locales/en.yml

```diff
@@ -108,6 +108,14 @@ en:
       to a growing catalogue of materials; for trainers, the portal offers an environment for sharing materials
       and event information; for trainees, it offers a convenient gateway via which to identify relevant training
       events and resources, and to perform specific, guided analysis tasks via customised training workflows.
+    mtessx: >
+      The <a href='https://elixirtess.github.io/mTeSS-X' target='_blank'>mTeSS-X Project</a> 
+      (Multi-space Training e-Support System with eXchange)
+      overcomes the fragmentation of training resources across Research Infrastructures and the European Science 
+      Clusters. The project aims to enhance existing TeSS-based training registries or catalogues like 
+      ELIXIR TeSS (life sciences) and 
+      <a href='https://www.panosc.eu/training-catalogue/'>PaN-Training (photon and neutron)</a>
```

Comment:
Can you also add `target='_blank'` to this link?

Reply:
Done.